### PR TITLE
feat: added UI for task list

### PR DESCRIPTION
**Pull request by aryan-13**
Changes Made: 
- Added UI for the taskList
- Added colours to the sidebar
- Completed the basic layout of the app

image

### CODE REVIEW

**Review by anshikaa12 — APPROVED** [review]

**Review by anshikaa12 — APPROVED** [review]
clean and well structured

File: src/Components/MainSection/TaskList/TaskList.jsx [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+	return (
+		<div className="main-section-container">
+			<input className="input-bar" placeholder="Click to quickly add a task" />
+			<button className="btn">Add</button>
+			<div className="task-list-container">
+				<div className="task-container">
+					<h2 className="task-heading">
+						Tasks <span className="task-count">3</span>
+					</h2>
+					<Task />
```
Comment (lannisterrr) [inline src/Components/MainSection/TaskList/TaskList.jsx:13]: try to map over the task instead of repeating the component.

**Review by lannisterrr — APPROVED** [review]
Good code structure , some suggesstion made.

File: src/Components/MainSection/TaskList/TaskList.jsx (hunk 1 above)
Comment (aryan-13) [reply src/Components/MainSection/TaskList/TaskList.jsx:13]: Yes, I'll map over the tasks, This is just for visualising the UI

**Pull request merged**